Project Requirements Document.md

```diff
@@ -0,0 +1,28 @@
+﻿
+|    **PROJECT REQUIREMENTS DOCUMENT**   |                                                                                                                                                                                                              |
+|:--------------------------------------:| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+|            **Project Name**            | Threewide                                                                                                                                                                                                    |
+|       **Functional Requirements**      |                                                                                                                                                                                                              |
+|            **High Priority**           | 1. Beginners can play Tetris game with modern tetris ruleset.                                                                                                                                                |
+|                                        | 2. Beginners can learn strategies to apply into game.                                                                                                                                                            |
+|                                        | 3. Our goal is to teach number of T spins through our plateform.                                                                                                                                             |
```

Comment:
It' s actually to teach T spins by asking the user to perform an n amount of them